src/ui/server/controllers/insurance/your-buyer/buyer-financial-information/index.ts

```diff
@@ -133,6 +135,13 @@ export const post = async (req: Request, res: Response) => {
       });
     }
 
+    // if no errors, then runs save api call
```

Comment:
Not a blocker but FYI the try/catch on such controllers only needs to start from just before the `mapAndSave` await call. I.e the try doesn't need to be at the top of the controller

Reply:
will do in next PR